### DISCUSSION TITLE
Python runtime into lib folder

### DISCUSSION THREAD

**Discussion by stef-pellegrino** [marcelotduarte/cx_Freeze#2400] [General]
Is it possible with actual version to set python3.dll and python32.dll into lib folder ?

Thanks

**Comment by marcelotduarte** [comment]
No, and I don't believe it's possible. I tried to use delayload previously but it wasn't possible either. I use delayload with cx_Logging as a dependency for Win32Service, so cx_Logging can be in the lib.
https://github.com/marcelotduarte/cx_Freeze/blob/main/setup.py#L299C36-L299C45

**Comment by stef-pellegrino** [comment]
Thanks for the answer (and for the tool ;))
